src/main/scala/cromwell/engine/backend/CromwellBackend.scala

```diff
@@ -0,0 +1,41 @@
+package cromwell.engine.backend
+
+import akka.actor.ActorSystem
+import org.slf4j.LoggerFactory
+import Backend.BackendyString
+
+/**
+  * Provides a global singleton access to the instantiated backend. This isn't bulletproof and can lead to
+  * exceptions being thrown but the way the system utilizes it it shouldn't be a problem (famous last words).
+  * So far the only real oddities I've seen are from the unit tests when they were trying to go straight to the
+  * interior of the system.
+  *
+  * Also we expect nearly everything to change about Backend in the next couple of months due to Intel's
```

Comment:
Probably an accurate statement (keeping in mind we don't *really* know what they have). I'll reword this all around as really I just meant the former bit. 

So for instance in the current world other obvious way to do this would be to string Backend around to the places that needed it, except that started spidering pretty extensively. Since we expect a lot of changes to what needs/expects/etc knowledge of backend throughout the non-backend portions it didn't make sense to go through that hassle (despite the fact that I already *did*, it was just really gross and so the singleton it was).

It might also turn out that nothing substantive changes, in which case we can re-evaluate the situation.